src/ComScoreStreamingAnalytics.ts

```diff
@@ -211,10 +230,23 @@ export class ComScoreStreamingAnalytics {
       ns_st_ia: this.metadata.advertisementLoad ? '1' : null,
       ns_st_cl: Math.round(assetLength * 1000),
     };
-
+    this.decorateUserContent(data);
     return data;
   }
 
+  private decorateUserContent(metadata: any): void {
```

Comment:
Not sure why we need this 🤔 

What about setting the `cs_ucfr: this.userConsent` directly in the `data` object?

Reply:
If the user hasn't explicitly granted or denied access, we do not want to include the `cs_ucfr` property in the `data` object

Reply:
Then this makes sense 👍